src/librustdoc/html/static/js/main.js

```diff
@@ -529,13 +529,15 @@ function preLoadCss(cssUrl) {
                 }
                 const link = document.createElement("a");
                 link.href = path;
-                if (path === current_page) {
-                    link.className = "current";
-                }
                 link.textContent = name;
                 const li = document.createElement("li");
                 li.appendChild(link);
                 ul.appendChild(li);
+                // Don't "optimize" this to just use `path`.
```

Comment:
Shouldn't this be done when generating the HTML instead?

Reply:
I forgot that we loaded modules with JS and yes, please move it before `ul.appendChild(li)` (better for my brain 😉 ).